services/controllerhost/config.go

```diff
@@ -49,15 +49,22 @@ type (
 	OutputPlacementConfig struct {
 		AdminStatus string `name:"adminStatus" default:"enabled"`
 	}
+
+	ExtentAssignmentConfig struct {
```

Comment:
This should really be ControllerDynamicConfig or simply ControllerConfig. When we start implementing other controller features (like placement / auto-scaling), those config params would also be part of this same config struct. The idea is to have one per service.